benches/payout-stakers/src/main.rs

```diff
@@ -30,23 +30,25 @@ const NOMINATE_CALL_BATCH_LIMIT: usize = 128;
 
 fn main() -> Result<(), anyhow::Error> {
     let address = "127.0.0.1:9944";
+    let ssl = false;
     let sudoer = AccountKeyring::Alice.pair();
 
     env_logger::init();
     info!("Starting benchmark with config ");
 
-    let connection = create_connection(address).set_signer(sudoer);
+    let connection = create_connection(address, ssl).set_signer(sudoer);
```

Comment:
Please consider using named function e.g. `create_connection_ws` or `create_connection_not_secured` here instead of bool param. Or name this param `not_secured` so it's more readable to see what type of connection we're creating here.

Reply:
solved in other thread